app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.java

```diff
@@ -14,11 +17,13 @@
 public class LoginFragment extends WolmoFragment<LoginPresenter> implements LoginView {
 
     private static LoginFragment instance;
+    private final int progress = 75;
```

Comment:
Which is the purpose of this int?

Reply:
I forgot to delete that :sweat_smile: I wanted to add more progress to the bar